genstack.yml

```diff
@@ -93,7 +93,7 @@ services:
             - sso.local.redhat.com
             - kcadmin
             - aabackend
-        command: /bin/bash -c "cd /app && npm install && ./wait_for_stack.sh && timeout -s SIGKILL 1000s cypress run --headless --browser chrome --spec cypress/integration/automation-analytics.js"
+        command: /bin/bash -c "cd /app && npm install && ./wait_for_stack.sh && timeout -s SIGKILL 1000s npm run integration_headless"
```

Comment:
There's no root `package.json` file in this repo so we'll need to hard-code the cypress command.

`/bin/bash -c "cd /app && npm install && ./wait_for_stack.sh && timeout -s SIGKILL 1000s cypress run --headless --browser chrome`